website/static/css/style.css

```diff
@@ -230,6 +230,46 @@ a {
     }
 }
 
+
+@media (max-width: 991px) {
```

Comment:
Bit concerned by arbitrary max widths, especially since the number of items on the nav bar is variable across projects. I'm not sure this will be applicable to preprints/registries.